app/renderer/css/main.css

```
@@ -170,6 +170,24 @@ body {
     border-radius: 4px;
 }
 
+.tab .server-icons:before { 
```

To be in coherence with the `alt-icon` design  of character the css of this class should be something like below.
Also there is a need to add the `not(.alt-icon)` selector so that the `before` element is not added to the elements which have already been taken care of by the `alt-icon` class and fixes.

```suggestion
.tab .server-icons:not(.alt-icon):before { 
    content: attr(data-content);
    font-family: Verdana;
    font-weight: 600;
    font-size: 22px;
    border: 2px solid #222c31;
    position: absolute;
    left: 0;
    width: 35px;
    margin-left: 17%;
    border-radius: 4px;
}
```